src/frontend/komponenter/Fagsak/Behandlingsresultat/behandlingsresultat.less

```diff
@@ -31,13 +31,13 @@
 
   & .typo-normal {
     &:first-child {
-      margin-top: 5.2rem;
+      margin-top: 4.8rem;
```

Comment:
Kan dette være et fint tidspunkt å gå bort fra less i denne filen? Planen er jo å gå helt bort fra det. 

Jeg synes fort det blir rotete når vi jobber på nøstede css-subklasser og det er litt vanskelig å se hva du endrer på her

Reply:
Dette brukes til fødselsnummer til venstre for radene i behandlingsresultat. Den er også gjennbrukt i tidslinje visualisering for labelen til venstre for radene der.

Reply:
Har flyttet CSS-en til styled components.